view/multi_match.js

```diff
@@ -0,0 +1,43 @@
+/**
+  this view is wrapped in a function so it can be re-used
+**/
+
+module.exports = function( properties ){
+  return function( vs ){
+
+    // base view
+    var view = { multi_match: {} };
+
+    properties.forEach(function(property) {
+      // validate required params
+      if( !property ||
+          !vs.isset('input:'+property) ||
+          !vs.isset('admin:'+property+':field') ){
+               return null;
+      }
+    });
+
+    if (!vs.isset('input:'+properties[0])) {
+      return null;
+    }
+
+    view.multi_match.analyzer = 'peliasAdmin';
```

Comment:
if you're going to hardcode the analyzer type here, we'd need to rename the file to indicate it's admin specific. otherwise, we'd have to allow the user to set the analyzer type in the config in the same place they specify the list of properties.